src/main/java/Duke/TaskList.java

```diff
@@ -0,0 +1,193 @@
+package Duke;
+import java.util.ArrayList;
+
+public class TaskList {
+    ArrayList<Task> list;
+    int curSize;
```

Comment:
Since calling `size()` on arrayList runs in constant time, is there any reason why you decided to store this in a separate variable?